src/conf_mode/vpn_openconnect.py

```diff
@@ -58,7 +58,7 @@ def get_config():
     default_values = defaults(base)
     ocserv = dict_merge(default_values, ocserv)
 
-    if "local" in ocserv["authentication"]["mode"]:
+    if 'mode' in ocserv["authentication"] and "local" in ocserv["authentication"]["mode"]:
```

Comment:
Can we use: `if "local" in dict_search("authentication.mode", ocserv):`?

Reply:
We don't have `mode` on this step